Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM python:3-alpine
 
+# Variables set with ARG can be overridden at image build time with
+# "--build-arg var=value".  They are not available in the running container.
+ARG B2_VERSION=v3.10.0
```

Comment:
```suggestion
ARG B2_VERSION=v3.19.1
```

Reply:
I don't see any changes that will affect our use of the `b2` command so I'll accept the suggestion.